suite-common/wallet-core/src/fiat-rates/fiatRatesSelectors.ts

```diff
@@ -27,21 +31,32 @@ import { FiatRatesRootState } from './fiatRatesTypes';
 
 type UnixTimestamp = number;
 
-export const selectFiatRates = (
-    state: FiatRatesRootState,
-    rateType: RateType = 'current',
-): FiatRates | undefined => state.wallet.fiat?.[rateType];
+export const selectCurrentFiatRates = (state: FiatRatesRootState): RatesByKey | undefined =>
+    state.wallet.fiat?.['current'];
+
+export const selectHistoricFiatRates = (state: FiatRatesRootState): RatesByTimestamps | undefined =>
+    state.wallet.fiat?.['historic'];
 
 export const selectFiatRatesByFiatRateKey = (
     state: FiatRatesRootState,
     fiatRateKey: FiatRateKey,
-    rateType: RateType = 'current',
+    rateType: RateTypeWithoutHistoric = 'current',
 ): Rate | undefined => state.wallet.fiat?.[rateType]?.[fiatRateKey];
 
+export const selectHistoricFiatRatesByTimestamp = (
+    state: FiatRatesRootState,
+    fiatRateKey: FiatRateKey,
+    timestamp: Timestamp,
+): number | undefined => {
+    const roundedTimestamp = roundTimestampToNearestPastHour(timestamp);
+
+    return state.wallet.fiat?.['historic']?.[fiatRateKey]?.[roundedTimestamp as Timestamp];
```

Comment:
This cast shall not be needed. The round function shall return casted type already